app/i18n/lv/gen.php

```diff
@@ -73,44 +73,44 @@
 		'Nov' => '\\N\\o\\v\\e\\m\\b\\r\\i\\s',
 		'Oct' => '\\O\\k\\t\\o\\b\\r\\i\\s',
 		'Sep' => '\\S\\e\\p\\t\\e\\m\\b\\r\\i\\s',
-		'apr' => 'Apr.',
+		'apr' => 'Apr.',	// IGNORE
 		'april' => 'Aprīlis',
-		'aug' => 'Aug.',
+		'aug' => 'Aug.',	// IGNORE
 		'august' => 'Augusts',
 		'before_yesterday' => 'Aizvakar',
-		'dec' => 'Dec.',
+		'dec' => 'Dec.',	// IGNORE
 		'december' => 'Decembris',
-		'feb' => 'Feb.',
+		'feb' => 'Feb.',	// IGNORE
 		'february' => 'Februāris',
-		'format_date' => 'j %s Y',
-		'format_date_hour' => 'j %s Y \\a\\t H\\:i',
+		'format_date' => 'j %s Y',	// IGNORE
+		'format_date_hour' => 'j %s Y \\a\\t H\\:i',	// IGNORE
 		'fri' => 'Pk',
-		'jan' => 'Jan.',
+		'jan' => 'Jan.',	// IGNORE
 		'january' => 'Janvāris',
 		'jul' => 'Jūlijs',
 		'july' => 'Jūlijs',
 		'jun' => 'Jūnijs',
 		'june' => 'Jūnijs',
-		'last_2_year' => 'Pēdējie divi gadi', // DIRTY: Not sure where this is used. Depending on the usage might need a different declension. Same goes for every last_* translation
+		'last_2_year' => 'Pēdējie divi gadi',
```

Comment:
This is used in the page for the statistics
https://github.com/FreshRSS/FreshRSS/blob/1ee2a3d72d97df34b5531ae6787f996ac8d62a2b/app/Controllers/statsController.php#L99-L126